src/bin/sol-fbp-generator/type-store.c

```diff
@@ -699,6 +699,23 @@ read_options(struct decoder *d, struct sol_vector *options)
 }
 
 static bool
+is_valid_symbol(const char *symbol)
+{
+    size_t i;
+
+    for (i = 0; symbol[i]; i++) {
+        if ((i == 0 && isdigit((int)symbol[i])) ||
+            (symbol[i] != '_' && !isalnum((int)symbol[i]))) {
+            fprintf(stderr, "Symbol '%s' with invalid character '%c' - pos: '%zu'\n",
```

Comment:
prefix with `ERROR: `? or some other we're using in the other error cases of this file?